src/cloudinventario_amazon_aws/collector.py

```diff
@@ -56,33 +56,54 @@ def _fetch(self, collect):
   def _get_instance_type(self, itype):
     if itype not in self.instance_types:
       types = self.client.describe_instance_types(InstanceTypes = [ itype ])
+      storage = self._get_storage_info()
       for rec in types['InstanceTypes']:
         name = rec['InstanceType']
         data = {
           "cpu": rec['VCpuInfo']['DefaultVCpus'],
           "memory": rec['MemoryInfo']['SizeInMiB'],
-          "storage": None,
-          "storages": []
+          "storage": storage['TotalSize'],
+          "storages": storage['Storages']
         }
         if rec.get("InstanceStorageInfo"):
           data["storage"] = rec['InstanceStorageInfo']['TotalSizeInGB'] * 1024,
           for disk in rec['InstanceStorageInfo']['Disks']:
             data['storages'].append({
               "id": None,
               "name": None,
-              "capacity": disk['SizeInGB'] * 1024,
+              "capacity": disk['SizeInGB'] * 1024,  # in MB
               "free": None,
               "type": disk['Type'],
-              "ssd": (disk['Type'] == 'ssd') or 0,
+              "ssd": (disk['Type'] == 'ssd') or 0
             })
```

Comment:
collection of these data was reimplmented in _get_storage_info, so this should have been removed